src/AutoRest.CSharp/Common/Generation/Writers/FormattableStringHelpers.cs

```diff
@@ -75,6 +75,9 @@ public static FormattableString GetIdentifiersFormattable(this IEnumerable<strin
 
         public static FormattableString? GetParameterInitializer(this CSharpType parameterType, Constant? defaultValue)
         {
+            if (parameterType.IsValueType)
+                return null;
```

Comment:
```suggestion
            if (parameterType.IsValueType)
            {
                return null;
            }
```

Looks like the other single-line `if` statements in this file are using brackets.   Plus, we're not monsters.